ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.13-nightly-x86.yaml

```diff
@@ -22,7 +22,7 @@ resources:
       memory: 200Mi
 tests:
 - as: perfscale-rosa-multiaz-control-plane-120nodes-periodic
-  cron: 0 12 * * 6
+  cron: 0 17 * * 4
```

Comment:
All three versions of rosa-multiaz 120nodes run on Thursday at 1700. I thought we want to stagger them more?